integration/spark/src/main/scala/org/apache/carbondata/spark/rdd/CarbonScanRDD.scala

```diff
@@ -1,311 +0,0 @@
-/*
```

Comment:
Better we just move the class to desired package and update it instead of removing the class and adding it in new package. It will maintain history.